tests/dragonfly/pymemcached_test.py

```diff
@@ -37,6 +37,19 @@ def test_mixed_reply(memcached_connection):
     assert memcached_connection.get(b"key") == b"data"
 
 
+@dfly_args({"memcached_port": 11211})
+def test_version(memcached_connection: pymemcache.Client):
+    """
+    php-memcached client expects version to be in the format of "n.n.n",
+    so we return 1.5.0 emulating an old memcached server. Our real version is being returned in the stats command. Also verified manually that php client parses correctly the version string
```

Comment:
My only concern here is that are you sure that this mocked version `1.5.0` is not used internally by the php client? 

What if the php client, uses this version to make some internal decision of some sort?

Reply:
I have not seen anything of that kind and I succeeded to run the client.